.github/workflows/codespace-test.yml

```
@@ -0,0 +1,40 @@
+name: Test in Codespace
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)